frontend/src/App.tsx

```diff
@@ -1,10 +1,14 @@
 import { Link, Route, Routes } from "react-router";
+import useInitializeGA from "./libs/googleAnalytics/hooks/useInitializeGA";
+import useTrackPageTimeGA from "./libs/googleAnalytics/hooks/useTrackPageTimeGA";
 import DetailPage from "./pages/detail/DetailPage";
 import ListPage from "./pages/list/ListPage";
 import Header from "./shared/components/Header/Header";
 import GlobalLayout from "./shared/components/Layout/GlobalLayout";
 
 function App() {
+  useInitializeGA();
+  useTrackPageTimeGA();
```

Comment:
```suggestion
  useInitializeGA();
  useTrackPageTimeGA();
```

`App` 컴포넌트의 관점에서는 GA의 어떤 작업들이 돌아가는지 몰라도 될 것 같아서 `useGA` 같은 훅으로 한번 더 추상화해서 관리하는 것도 괜찮을 것 같다는 생각이 드는데 세오의 생각은 어떠신가요??

`App` 컴포넌트: GA가 동작한다는 것만 알면 된다.
`useGA` 훅: GA의 어떤 작업들이 동작하는지 알아야 한다.

Reply:
동의보감입니다.

Reply:
GA 초기화 방식에 대해 조금 고민해보다가, 아래와 같은 구조를 한 번 제안드려봅니다🙇‍♂️

- 초기화는 `GAInitializer`라는 전용 컴포넌트를 만들어 `main.tsx`에서 마운트하는 방식으로 처리하면 어떨까 합니다.
이렇게 하면 앱 실행 시점에 딱 한 번만 초기화되도록 보장할 수 있고, 책임도 명확해지는 느낌이었습니다.
- 페이지별 체류 시간 측정은 현재처럼 `App.tsx`에서 `useTrackPageTimeGA()` 훅을 사용하는 식으로 유지하면, 라우팅과 자연스럽게 연동될 수 있다고 생각했습니다.

초기화와 트래킹을 나누는 게 과한 분리일 수도 있어서, 혹시 이 방식에 대한 의견이나 더 나은 구조가 있으시면 말씀 부탁드립니다! 🙏  
다른 분들 의견 듣고 조율 및 결정을 하려고 합니다.
다른 의견이 있으실 경우 언제든지 편하게 말씀해주시면 감사하겠습니다🙇‍♂️

Reply:
좋은 것 같습니다~!👍👍👍

Reply:
과한 분리로는 생각되지 않는 것 같아요!